web/static/js/thesis.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```

Comment:
This is now the main file that builds Thesis and attaches a constructor function to the `window` object as `window.thesis`.